Framework/PythonInterface/test/python/plugins/algorithms/WorkflowAlgorithms/CorelliPowderCalibrationCreateTest.py

```diff
@@ -34,16 +34,16 @@ def test_exec(self):
         # centers for all detectors in the bank (not just detector-ID 155) approach our reference values. As
         # a result, the final position and orientation is not exactly perpendicular to the X-axis and positioned
         # five meters away from the sample.
-        target_position, target_orientation, target_rotation = [5.18, -0.32, 0.21], [0.001, 0.999, -0.027], 98.0
+        target_position, target_orientation, target_rotation = [5.18, -0.32, 0.20], [0.001, 0.999, -0.027], 98.0
         CorelliPowderCalibrationCreate(
             InputWorkspace='test_workspace', OutputWorkspacesPrefix='cal_', TubeDatabaseDir='/tmp',
             TofBinning=[300, 1.0, 16666.7], PeakPositions=spacings_reference, AdjustSource=False, ComponentList='bank1',
             ComponentMaxTranslation=0.2, ComponentMaxRotation=10)
         row = mtd['cal_adjustments'].row(0)
-        assert_allclose([row[name] for name in ('Xposition', 'Yposition', 'Zposition')], target_position, atol=0.004)
+        assert_allclose([row[name] for name in ('Xposition', 'Yposition', 'Zposition')], target_position, atol=0.05)
```

Comment:
The absolute tolerance here is much bigger than before. Is this relaxation necessary?

Reply:
The different library versions installed on the different operative systems are yielding slightly different optimized positions/orientations for the banks. My guess is that the new objective function is "easier" to minimize so that slightly different positions/orientations yield similar peak-center deviations from the reference d-spacings.

Reply:
This is kind of strange :thinking: .
It almost sounds like the solution is unstable.
Maybe we should leave a comment here so that we can revisit this part once we have more time.

Reply:
Sounds good to me. If we ever find a problem with this algorithm, we'll have a suspect area to start to looking

Reply:
I inserted a ToDo